prow/github/types.go

```diff
@@ -1090,14 +1101,70 @@ type Milestone struct {
 // Note that it's wrapping a GitCommit, so author/committer information is in two places,
 // but contain different details about them: in RepositoryCommit "github details", in GitCommit - "git details".
 type RepositoryCommit struct {
-	SHA         string    `json:"sha"`
-	Commit      GitCommit `json:"commit"`
-	Author      User      `json:"author"`
-	Committer   User      `json:"committer"`
-	Parents     []Commit  `json:"parents,omitempty"`
-	HTMLURL     string    `json:"html_url"`
-	URL         string    `json:"url"`
-	CommentsURL string    `json:"comments_url"`
+	NodeID      string   `json:"node_id,omitempty"`
+	SHA         string   `json:"sha,omitempty"`
+	Commit      Commit   `json:"commit,omitempty"`
+	Author      User     `json:"author,omitempty"`
+	Committer   User     `json:"committer,omitempty"`
+	Parents     []Commit `json:"parents,omitempty"`
+	HTMLURL     string   `json:"html_url,omitempty"`
+	URL         string   `json:"url,omitempty"`
+	CommentsURL string   `json:"comments_url,omitempty"`
+
+	// Details about how many changes were made in this commit. Only filled in during GetCommit!
+	Stats CommitStats `json:"stats,omitempty"`
```

Comment:
Please make this a pointer, otherwise it will always be present

Reply:
Fixed.

Reply:
Fixed.